.github/workflows/deploy-lambda-code.yaml

```diff
@@ -40,6 +54,52 @@ on:
         type: string
 
 jobs:
+  derive-service-names:
```

Comment:
see comment above about whether services can be anything other than all

Reply:
We had and want to retain the ability to cherry pick services to deploy (in dev/test and maybe prod etc) but in integration because this deployment is part of a matrix step it will always be set to all